sdk/python/kfp/cli/component.py

```diff
@@ -172,9 +172,9 @@ def _load_components(self):
 
                 python_file = str(python_file)
                 if not component_modules:
-                    logging.error(
```

Comment:
nit/question: I'm curious what this logging adds without the check.

I think the logging on line 175 is now redundant with the logging on line 179, since the execution will not `sys.exit` anymore. WDYT about either [simpler approach] removing the entire `if not component_modules` block or [more involved approach] changing the logging to something like the following:
```
comp_dir/file.py
comp_dir/other_file.py
comp_dir/component.py [component]
```

I'm not sure `[component]` is the best way of visually displaying this information, but just thinking out loud with this mock up.

Reply:
Good point! I'll opt for removing this if block.